guides/Mix Dependencies.md

```diff
@@ -0,0 +1,26 @@
+# Mix Dependencies
+
+Many harness packages write a `mix.exs` file which controls compilation,
+configuration, testing, and dependency options.
+
+## Installing/updating relay_commands and/or events
+
+Instead of adding the events and relay_commands dependencies to the `mix.exs` 
+file like we would normally do, we add it to the list of `explicit_dependencies` 
+in the `harness.exs` file.
+
+Example:
+```
+config :harness_aggregate,
+  app: :planning_schedule,
+  aggregate_type: "planning.schedule",
```

Comment:
```suggestion
  aggregate_type: "Planning.Schedule",
```

you probably wanna change this in the Planning.Schedule repo as well

should be fixed in the gaas generator to enforce this formatting but /shrug that takes work. I'll get around to it eventually